app/src/main/java/com/a5corp/weather/SplashScreen/SplashScreen.java

```diff
@@ -0,0 +1,46 @@
+package com.a5corp.weather.SplashScreen;
```

Comment:
Please go through the code of the app. Since `activity` sub-package already exists, please place this activity in that sub-package and not create your own package